src/modules/vmount/output.h

```diff
@@ -52,6 +48,7 @@
 namespace vmount
 {
 
+#if 0
```

Comment:
?

Reply:
I'll remove it. This is no longer required as I'm injecting the structure with all params instead.